src/shared/types.ts

```diff
@@ -1,17 +1,23 @@
 export type UserReport = {
-  uuid: string;
-  reported_at: Date;
-  url: string;
+  app_channel: string;
```

Comment:
Sorry for the weird diff here. I decided to sort the values alphabetically, because that's what I usually like. But it wasn't sorted that way previously, so it got a bit confusing.